constraint/pkg/instrumentation/instrumentation.go

```diff
@@ -0,0 +1,54 @@
+// package instrumentation defines primitives to
+// support more observability throughout the constraint framework.
+package instrumentation
+
+const (
+	// scope constants.
+	TemplateScope = "template"
+
+	// description constants.
+	UnknownDescription = "unknown description"
+
+	// source constants.
+	EngineSourceType = "engine"
+	RegoEngineSource = "rego"
```

Comment:
This value should be defined by the engine, it should not live elsewhere in the code. Currently defined here:

https://github.com/open-policy-agent/frameworks/blob/745f2b50cfcb746bb30997d65e851d1b6ee0b9e9/constraint/pkg/client/drivers/rego/schema/schema.go#L12